FAQ.md

```diff
@@ -242,6 +242,7 @@ Possible reasons:
 - If you run multiple DDL statements together, the last few DDL statements might run slowly. This is because the DDL statements are executed serially in the TiDB cluster.
 - After you start the cluster successfully, the first DDL operation may take a longer time to run, usually around 30s. This is because the TiDB cluster is electing the leader that processes DDL statements.
 - In rolling update or shutdown update, the processing time of DDL statements in the first ten minutes after starting TiDB is affected by the server stop sequence (stopping PD -> TiDB), and the condition where TiDB does not clean up the registration data in time because TiDB is stopped using the `kill -9` command. When you run DDL statements during this period, for the state change of each DDL, you need to wait 2 * lease (lease = 10s).
+- When a communication problem occurs between a TiDB server and a PD server in the cluster, or in other words, the TiDB server can not get or update the version information from the PD server in time, for the state processing of each DDL, you need to wait 2 * lease.
```

Comment:
This sentence is too long.
-> If a communication issue occurs between a TiDB server and a PD server in the cluster, the TiDB server cannot get or update the version information from the PD server in time. In this case, you need to wait for 2 * lease for the state processing of each DDL.

Reply:
Please also change the previous line "you need to wait 2 * lease (lease = 10s)" to "you need to wait for 2 * lease (lease = 10s)".